### ISSUE TITLE
Angular 6 error in rxjs

### ISSUE THREAD

**Issue by vh13294** [abhazelton/animate-on-scroll#18]
After importing this module to my project, I recieve this error!
from this line << var fromEvent_1 = require("rxjs/observable/fromEvent"); >>

ERROR in ./node_modules/ng2-animate-on-scroll/dist/src/scroll.service.js
Module not found: Error: Can't resolve 'rxjs/observable/fromEvent' in '/home/panha/Documents/portfolio/node_modules/ng2-animate-on-scroll/dist/src'

**Comment by vh13294** [comment]
I am currently on version 1.0.11, but there is still an error. 
However, I can easily fix this on my local machine, but not the npm build.

**Comment by abhazelton** [comment]
@vh13294 I meant your installed version of rxjs.

**Comment by vh13294** [comment]
Yeah I am currently running "rxjs": "^6.3.3" on lastest angular-cli.
But the problem still persist.

**Comment by abhazelton** [comment]
Glad you got it!
